src/problem.jl

```diff
@@ -72,13 +72,22 @@ function JumpProblem(prob, aggregator::AbstractAggregatorAlgorithm, jumps::JumpS
                      save_positions = typeof(prob) <: DiffEqBase.AbstractDiscreteProblem ? (false,true) : (true,true),
                      rng = Xorshifts.Xoroshiro128Star(rand(UInt64)), kwargs...)
 
+  # initialize the MassActionJump rate constants with the user parameters
```

Comment:
@ChrisRackauckas this aliases all the non-rate constant fields while deferring constructing the rates until `JumpProblem`. Prior to that I just set the rate field to be `nothing` as you suggested.